Code.js

```diff
@@ -5,16 +5,35 @@ const form = FormApp.openById(formId);
 const qrCodeFileId = "1BCo66b-O-ablSuihrBZU1hpc1Kb-woso"
 const qrCodeBlob = DriveApp.getFileById(qrCodeFileId).getAs('image/jpeg')
 
+// new student response (google form) related - col number starts from 0
+const studentNameColNum = 1;
+const studentWechatColNum = 2;
+const studentOtherContactColNum = 3;
+const timeColNum = 4;
+const timeConfColNum = 5;
+
 function bind(){
     ScriptApp.newTrigger('myOnFormSubmit')
     .forForm(form)
     .onFormSubmit()
     .create();
 }
 
+
 function myOnFormSubmit(event) {
   let email = event.response.getRespondentEmail();
-  sendConfirmationEmail(email);
+  let formRespones = event.response.getItemResponses();
+  let studentTimeCol_1 = formRespones[timeColNum].getResponse();
+  let studentTimeCol_2 = formRespones[timeConfColNum].getResponse();
+
+  // If time entered twice is different, send time conflict failure email
+  // Otherwise, send confirmation email
+  if (studentTimeCol_1 != studentTimeCol_2){
```

Comment:
if we decide to migrate to namedValues then this line is fine. Right now it will not behave as expected because we are comparin two response objects, instead of their values.